modules/oadp-installing-dpa-1-3.adoc

```diff
@@ -0,0 +1,392 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-installing-dpa-1-3_{context}"]
+= Installing the Data Protection Application 1.3
+
+You install the Data Protection Application (DPA) by creating an instance of the `DataProtectionApplication` API.
+
+.Prerequisites
+
+* You must install the OADP Operator.
+* You must configure object storage as a backup location.
+* If you use snapshots to back up PVs, your cloud provider must support either a native snapshot API or Container Storage Interface (CSI) snapshots.
+* If the backup and snapshot locations use the same credentials, you must create a `Secret` with the default name, `{credentials}`.
+ifdef::installing-oadp-azure,installing-oadp-gcp,installing-oadp-mcg,installing-oadp-ocs,virt-installing-configuring-oadp[]
+* If the backup and snapshot locations use different credentials, you must create two `Secrets`:
+
+** `Secret` with a custom name for the backup location. You add this `Secret` to the `DataProtectionApplication` CR.
+** `Secret` with the default name, `{credentials}`, for the snapshot location. This `Secret` is not referenced in the `DataProtectionApplication` CR.
```

Comment:
Ever since oadp 1.2, secret for snapshotlocation can be custom name as well.
https://github.com/openshift/velero/commit/596114b4279d8b8472d9f3bc1f8967df17186efe

cc @sseago

Reply:
@kaovilai That's right. Either snapshot location or backup location can use default secret. Either snapshot location or backup location can use custom secret.

Reply:
@kaovilai @sseago 
If there is something you want me to change, please explain exactly what you want changed.

Reply:
I guess this is fine as is.. otherwise following could work
```suggestion
** `Secret` with another custom name for the snapshot location. You add this `Secret` to the `DataProtectionApplication` CR.
```

Reply:
@kaovilai 
I made the change in the 1.3 and 1.2 DPAs